databricks/koalas/ml.py

```diff
@@ -43,13 +54,13 @@ def corr(kdf, method='pearson'):
     return arr
 
 
-def to_numeric_df(kdf):
+def to_numeric_df(kdf: 'ks.DataFrame') -> Tuple[pyspark.sql.DataFrame, List[str]]:
     """
     Takes a dataframe and turns it into a dataframe containing a single numerical
     vector of doubles. This dataframe has a single field called '_1'.
```

Comment:
same here ... an example would be great

Reply:
That function is more meant to be internal